Lib/pydoc_data/topics.py

```diff
@@ -13174,8 +13174,7 @@ class dict(iterable, /, **kwargs)
 
 See also:
 
-  "types.MappingProxyType" can be used to create a read-only view of a
-  "dict".
+  "frozendict" can be used to create a read-only view of a "dict".
```

Comment:
With "view" do you mean that no copy is taking place? So that if I modify the underlying dict, the read-only view will also appear updated?

I think that's not the case, and we should rephrase this
```suggestion
  "frozendict" can be used to create a read-only copy of a "dict".
```